docs/contributing/code_contribution_guide.rst

```diff
@@ -143,25 +156,15 @@ Once you have synced your branch push your changes and check if file conflicts a
 General Tips
 -------------
 
-1. It's good practice to link PR to an issue during commit message. Such as
-stating ``Fix #132`` for fixing issue 132.
-
-2. If you have an issue, ask your question in slack before reporting issue. If
-your issue is not resolved check any open issues for resolution before creating
-a new issue.
+- If you have an issue, ask your question in slack before reporting issue. If your issue is not resolved check any open issues for resolution before creating a new issue.
```

Comment:
Please add "the" after "before reporting"

Reply:
fixed